deco-sites/otica-isabela
# applying styles into product details page

## What is this contribution about?

Changing product page to be with the proper expected layout.

## How to test it?

Access the store and go to any product, check it on views: desktop, tablet and mobile.
